src/server/utilities/cspHeader/index.js

```diff
@@ -230,6 +232,10 @@ const directives = {
       'https://www.instagram.com', // Social Embeds
       'http://www.instagram.com', // Social Embeds
       'https://cdn.syndication.twimg.com', // Social Embeds
+      'https://static.stage.bbc.co.uk', // STY include
+      'http://static.stage.bbc.co.uk', // STY include
```

Comment:
Where is stage being used?

Reply:
it's being used in the include we have locally

Reply:
Is that a 'real' include or one we have created/modified? Please can you link to it so I can look? As with test above, we should not have any non-live origins in the live lists.

Reply:
I just confirmed it's an include that's being modified.